docs/rules/replace-j-unit4-assumptions-with-jupiter.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Replace JUnit 4 Assumptions with JUnit Jupiter
+ruleId: ReplaceJUnit4AssumptionsWithJupiter
+since: 3.30.0
+minJavaVersion: 8
+remediationCost: 2
+    
+description:
+    This rule contributes to the stepwise transition from JUnit 4 to JUnit 5 by replacing the assumption methods defined in the JUnit 4 class 'org.junit.Assume' by equivalent assumption methods defined in the JUnit 5 class 'org.junit.jupiter.api.Assumptions'.
+tags: ["Java 8", "Testing"]
+---
+
+# Replace JUnit 4 Assumptions with JUnit Jupiter
+
+## Description
+
+This rule contributes to a stepwise transition to JUnit 5 by replacing the JUnit 4 assumption methods by the equivalent JUnit 5 ones.
+In JUnit 5, all methods used to carry out assumptions are declared in [`org.junit.jupiter.api.Assumptions`](https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/Assumptions.html), while in JUnit 4 the equivalent methods are declared in [`org.junit.Assume`](https://javadoc.io/doc/junit/junit/latest/org/junit/Assume.html).
+
+This rule looks for invocations of overloaded methods of the JUnit 4 class `Assume`, for example:
+* [`assumeFalse`](https://javadoc.io/doc/junit/junit/latest/org/junit/Assume.html#assumeFalse(boolean))
+* [`assumeTrue`](https://javadoc.io/doc/junit/junit/latest/org/junit/Assume.html#assumeTrue(boolean))
```

Comment:
Please note, we can put these on a table. You can have a look at replace-j-unit4-assertions-with-jupiter.